developer/messages/page-is-not-editable.mdx

```diff
@@ -0,0 +1,24 @@
+---
+# this page is referenced in Leo, please do not rename without updating the reference in:
+# https://github.com/makeswift/cosmos/blob/master/packages/leo/src/lib/docs.ts
+
+title: This page is not editable. There are no available elements or page props to edit.
```

Comment:
What are "page props" here? This seems like a concept that we don't mention in our docs or talk about and I would fine confusing if it's my first time hearing of it.

Reply:
Good point. I think this was wording from [our design spec](https://www.figma.com/design/DUBoFzjgADywP2h8ZXLmyo/App-v2?node-id=17047-196718&t=C6JjYxT6oE40zjTu-4). cc @andrewreifman 

@migueloller What we should say? Maybe something like this?

> This page is not editable.
> There are no available elements to edit or the page is not integrated with Makeswift.

Reply:
Yeah, I think that's a good alternative 👍🏻